crates/pop-cli/src/commands/build/mod.rs

```diff
@@ -35,13 +35,9 @@ pub(crate) struct BuildArgs {
 	/// Build profile [default: debug].
 	#[clap(long, value_enum)]
 	pub(crate) profile: Option<Profile>,
-	/// Parachain ID to be used when generating the chain spec files.
-	#[arg(short = 'i', long = "id")]
-	#[cfg(feature = "parachain")]
-	pub(crate) id: Option<u32>,
 }
 
-/// Build a parachain, smart contract or Rust package.
+/// Build a parachain, smart contract, chain specification or Rust package.
```

Comment:
This comment is incorrect as it only provides the possibility for building a chain specification. The others are not included here as they fall under the BuildArgs

Reply:
It doesn't show this in the `--help` anymore, but you are right: _Build a chain specification and its genesis artifacts._